docs/conf.py

```diff
@@ -59,6 +59,8 @@
      'One line description of project.',
      'Miscellaneous'),
 ]
+# -- Options for images ------------------------------------------------
+--image-loading = 'lazy'
```

Comment:
```suggestion
image-loading = 'lazy'
```

Not sure but as this is Python - I think the dash prefix will not work. The ci seems to have failed.

Reply:
Even this will not work I think, did you run this locally?

Reply:
I think it should be `image_loading = 'lazy'`